addons/htpasswd-identity-provider/htpasswd-identity-provider.addon.remove

```diff
@@ -1,20 +1,32 @@
+ssh sudo cp -fp #{MINISHIFT_DATA_HOME}/#{CONFIG_LOCATION_OPENSHIFT_API}/#{MASTER_CONFIG_FILE} #{MINISHIFT_DATA_HOME}/#{CONFIG_LOCATION_OPENSHIFT_API}/master-config-htpasswd.yaml
+ssh sudo cp -fp #{MINISHIFT_DATA_HOME}/#{CONFIG_LOCATION_KUBERNETES_API}/#{MASTER_CONFIG_FILE} #{MINISHIFT_DATA_HOME}/#{CONFIG_LOCATION_KUBERNETES_API}/master-config-htpasswd.yaml
 
-# backup existing configuration
-ssh sudo cp -fp #{MINISHIFT_DATA_HOME}/#{CONFIG_LOCATION}/#{MASTER_CONFIG_FILE} #{MINISHIFT_DATA_HOME}/#{CONFIG_LOCATION}/master-config-prepatch.yaml
 # revert to AllowAllPasswordIdentityProvider
-ssh sudo #{MINISHIFT_DATA_HOME}/bin/oc ex config patch #{MINISHIFT_DATA_HOME}/#{CONFIG_LOCATION}/master-config-prepatch.yaml --patch='{"oauthConfig": {"identityProviders": [ {"challenge": true,"login": true,"mappingMethod": "claim","name": "anypassword","provider": {"apiVersion": "v1","kind": "AllowAllPasswordIdentityProvider"}}]}}' > #{MINISHIFT_DATA_HOME}/#{CONFIG_LOCATION}/master-config.yaml
-# remove the prepatch file
-ssh sudo rm -f #{MINISHIFT_DATA_HOME}/#{CONFIG_LOCATION}/master-config-prepatch.yaml
+ssh sudo #{MINISHIFT_DATA_HOME}/bin/oc ex config patch #{MINISHIFT_DATA_HOME}/#{CONFIG_LOCATION_OPENSHIFT_API}/master-config-htpasswd.yaml --patch='{"oauthConfig": {"identityProviders": [ {"challenge": true,"login": true,"mappingMethod": "claim","name": "anypassword","provider": {"apiVersion": "v1","kind": "AllowAllPasswordIdentityProvider"}}]}}' > #{MINISHIFT_DATA_HOME}/#{CONFIG_LOCATION_OPENSHIFT_API}/master-config.yaml
+ssh sudo #{MINISHIFT_DATA_HOME}/bin/oc ex config patch #{MINISHIFT_DATA_HOME}/#{CONFIG_LOCATION_KUBERNETES_API}/master-config-htpasswd.yaml --patch='{"oauthConfig": {"identityProviders": [ {"challenge": true,"login": true,"mappingMethod": "claim","name": "anypassword","provider": {"apiVersion": "v1","kind": "AllowAllPasswordIdentityProvider"}}]}}' > #{MINISHIFT_DATA_HOME}/#{CONFIG_LOCATION_KUBERNETES_API}/master-config.yaml
+
+
 # remove users.htpasswd file 
-ssh sudo rm -f #{MINISHIFT_DATA_HOME}/#{CONFIG_LOCATION}/users.htpasswd
+ssh sudo rm -f #{MINISHIFT_DATA_HOME}/#{CONFIG_LOCATION_OPENSHIFT_API}/users.htpasswd
+ssh sudo rm -f #{MINISHIFT_DATA_HOME}/#{CONFIG_LOCATION_KUBERNETES_API}/users.htpasswd
 
-# restart openshift 
-docker stop origin 
-docker start origin
+echo -- Restarting Openshift API server ..
+docker stop $(docker ps -l -q --filter "label=io.kubernetes.container.name=apiserver")
+echo -- Restarting Kube API server ..
+docker stop $(docker ps -l -q --filter "label=io.kubernetes.container.name=api")
+
+ssh until curl -f -k https://#{ip}:8443/healthz;do sleep 1;done
+
+# remove the prepatch file
+ssh sudo rm -f #{MINISHIFT_DATA_HOME}/#{CONFIG_LOCATION_OPENSHIFT_API}/master-config-htpasswd.yaml
+ssh sudo rm -f #{MINISHIFT_DATA_HOME}/#{CONFIG_LOCATION_KUBERNETES_API}/master-config-htpasswd.yaml
 
-echo 'Successfully removed addon htpasswd-identity-provider'
+echo -- Successfully removed addon htpasswd-identity-provider ... OK
```

Comment:
`htpasswd` is not removed from the filesystem...

Reply:
@gbraad All done now.